src/System.IO.Pipes/src/System.IO.Pipes.csproj

```diff
@@ -132,11 +129,10 @@
     <Compile Include="Microsoft\Win32\SafeHandles\SafePipeHandle.Windows.cs" />
     <Compile Include="System\IO\Pipes\AnonymousPipeServerStream.Windows.cs" />
     <Compile Include="System\IO\Pipes\ConnectionCompletionSource.cs" />
-    <Compile Include="System\IO\Pipes\IOCancellationHelper.Windows.cs" />
     <Compile Include="System\IO\Pipes\NamedPipeClientStream.Windows.cs" />
     <Compile Include="System\IO\Pipes\NamedPipeServerStream.Windows.cs" />
     <Compile Include="System\IO\Pipes\PipeStream.Windows.cs" />
-    <Compile Include="System\IO\Pipes\PipeStreamAsyncResult.cs" />
+    <Compile Include="System\IO\Pipes\PipeStreamCompletionSource.cs" />
```

Comment:
Could we name the file and type something more descriptive, like ReadWriteCompletionSource, or something like that?